stock_request/tests/test_stock_request.py

```diff
@@ -734,7 +734,7 @@ def test_cancel_request(self):
         order = self.request_order.with_user(self.stock_request_user).create(vals)
 
         self.product.route_ids = [(6, 0, self.route.ids)]
-        order.with_user(self.stock_request_user).action_confirm()
+        order.with_user(self.stock_request_manager).action_confirm()
```

Comment:
why is this needed? test were working before this improvement, what have changed?

Reply:
Not in this module, but stock_request_user don't belong to inventory users, so for example when mrp is installed and tests run I've got several errors regarding read access that cannot be avoidable during installation adding access rules on extensible module.

Reply:
fair enough